# day_20200114 总结

## 谷歌报错

```
Unchecked runtime.lastError: Could not establish connection. Receiving end does not exist.
```

拓展程序出错